CHANGELOG.md

```diff
@@ -95,6 +129,38 @@ The library has been expanded to include the following services:
   retired. If you are affected by this removal, please use
   `Client::GetNativeBucketIamPolicy` and `Client::SetNativeBucketIamPolicy`
   instead. See [#5929] for more details.
+* cleanup(common)!: remove legacy IAM classes ([#8667](https://github.com/googleapis/google-cloud-cpp/pull/8667))
+* cleanup(storage)!: remove deprecated IAM functions ([#8664](https://github.com/googleapis/google-cloud-cpp/pull
+* cleanup!: change storage bazel visibility to private by default ([#8583](https://github.com/googleapis/google-cloud-cpp/pull/8583))
+
+**Other Changes**
+
+* doc(storage): add custom-dual regions example ([#8595](https://github.com/googleapis/google-cloud-cpp/pull/8595))
+/8664))
+* fix(storage): `ParallelUploadFile()` and `WithObjectMetadata` ([#8629](https://github.com/googleapis/google-cloud-cpp/pull/8629))
+* fix(storage): return an error on short writes ([#8562](https://github.com/googleapis/google-cloud-cpp/pull/8562))
+* fix(storage): use correct offset and size types ([#8538](https://github.com/googleapis/google-cloud-cpp/pull/8538))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+**BREAKING CHANGES**
+
+* cleanup(common)!: remove legacy IAM classes ([#8667](https://github.com/googleapis/google-cloud-cpp/pull/8667))
+
+**Other Changes**
+
+* fix(pubsublite): internal namespaces ([#8680](https://github.com/googleapis/google-cloud-cpp/pull/8680))
```

Comment:
Should we just remove everything about pubsublite?  Or should it at least go under the (previously used) "Pub/Sub Lite" banner?